CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+Version 1.16.1
+==============
+**June 2, 2023**
+
+This release contains a small number of enhancements and important bug fixes. Many thanks to @philippjfr
+and @Hoxbro.
+
+This release includes a breaking change as notebooks will no longer be inlining as default. This change will reduce the size of the notebook files and, more importantly, address an upstream issue where running `hv.extension` would give an error when used in a notebook.
+
+
+Enhancements:
```

Comment:
I would put the section where you describe the fix for `inline=True` before the enhancements section.